types/ismobilejs/index.d.ts

```diff
@@ -3,41 +3,55 @@
 // Definitions by: Maksim Karelov <https://github.com/Ty3uK>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export const any: boolean;
-export const phone: boolean;
-export const tablet: boolean;
-export const seven_inch: boolean;
-
-export const apple: {
-    phone: boolean;
-    ipod: boolean;
-    tablet: boolean;
-    device: boolean;
-};
-
-export const android: {
-    phone: boolean;
-    tablet: boolean;
-    device: boolean;
-};
-
-export const amazon: {
-    phone: boolean;
-    tablet: boolean;
-    device: boolean;
-};
-
-export const windows: {
-    phone: boolean;
-    tablet: boolean;
-    device: boolean;
-};
-
-export const other: {
-    blackberry_10: boolean;
-    blackberry: boolean;
-    opera: boolean;
-    firefox: boolean;
-    chrome: boolean;
-    device: boolean;
-};
+interface Apple {
+  phone: boolean;
+  ipod: boolean;
+  tablet: boolean;
+  device: boolean;
+}
+
+interface Android {
+  phone: boolean;
+  tablet: boolean;
+  device: boolean;
+}
+
+interface Amazon {
+  phone: boolean;
+  tablet: boolean;
+  device: boolean;
+}
+
+interface Windows {
+  phone: boolean;
+  tablet: boolean;
+  device: boolean;
+}
+
+interface Other {
+  blackberry_10: boolean;
+  blackberry: boolean;
+  opera: boolean;
+  firefox: boolean;
+  chrome: boolean;
+  device: boolean;
+}
+
+interface IsMobile {
+  any: boolean;
+  phone: boolean;
+  tablet: boolean;
+  seven_inch: boolean;
+
+  apple: Apple;
+  android: Android;
+  amazon: Amazon;
+  windows: Windows;
+  other: Other;
+
+  (userAgent?: string): IsMobile;
+}
+
+declare const isMobile: IsMobile;
+
+export default isMobile;
```

Comment:
I think this should be `export =` an not `export default`, because `require("ismobilejs").default` is undefined. See the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).

Reply:
@andy-ms thanks!